core/src/test/java/org/hibernate/ogm/test/cascade/CascadeEntityLoaderTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.test.cascade;
```

Comment:
Shouldn't this test run for all the datastores?

Reply:
Yes, it should. I'm going to move the package.